Cubical/Foundations/GroupoidLaws.agda

```diff
@@ -0,0 +1,93 @@
+{-
+
+This file proves the higher groupoid structure of types
+
+-}
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Foundations.GroupoidLaws where
+
+open import Cubical.Core.Primitives
+open import Cubical.Core.Prelude
+
+private
+  variable
+    ℓ : Level
+    A : Set ℓ
+
+-- some useful notation
+
+_·_ :  {x y z : A} → (x ≡ y) → (y ≡ z) → (x ≡ z)
+x≡y · y≡z = compPath x≡y y≡z 
```

Comment:
This should be upstreamed and be the definition of compPath. How about doing it in this PR? (The font in my browser is a bit weird, but to input this symbol you use `\.` right?)

Reply:
> This should be upstreamed and be the definition of compPath. How about doing it in this PR? (The font in my browser is a bit weird, but to input this symbol you use `\.` right?)

I think he used `\cdot`. I prefer `\.` too.

Reply:
> This should be upstreamed and be the definition of compPath. How about doing it in this PR?

Do you mean it should be definition or notation of compPath?

>(The font in my browser is a bit weird, but to input this symbol you use `\.` right?)

I used `\cdot`, but I don't have any particular attachment to it. Let me change it to `\.`.

Reply:
It should be the definition of compPath